packages/opentelemetry-core/test/trace/fixtures/test-package/package.json

```diff
@@ -4,6 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "compile": "tsc --build",
```

Comment:
nothing against we don't think its usefull to change this one :)

Reply:
Overzealous find and replace it looks like